packages/paste-website/src/pages/customization/customizing-component-elements.mdx

```diff
@@ -33,7 +33,179 @@ export const getStaticProps = async () => {
 
 <content>
 
-We are currently working on this feature and providing support for it gradually across the entire suite of components. Watch this space!
+As a designer or developer of a Twilio Programmable App, you may need to customize a component to match your specific design requirements. Use Paste’s Customization Provider API to customize individual components or create entirely new ones based on existing components.
+
+The Customization Provider seamlessly integrates with the current theme so that your customizations work perfectly with the default or dark base themes, or any fully custom theme you provide. To make that as easy as possible for you, we have taken the decision to provide some guardrails with that API, only allowing design token values on certain CSS properties where it makes sense.
+
+## Understand the element prop and data attributes
+
+Each Paste component has a collection of tagged DOM nodes within it, each with a `data-paste-element` data attribute that identifies each stylable element you can target. The `element` prop on each Paste component can also be used to change or override the element name for that particular instance of the component. When this prop is set, it renders a custom `data-paste-element` data attribute extending the base Paste component to being something completely new.
+
+For example, the following code will render a custom neutral alert with the element name `"MY_CUSTOM_ALERT"`:
+
+```jsx
+export const App = () => {
+  return (
+    <CustomizationProvider
+      elements={{
+        MY_CUSTOM_ALERT: {
+          paddingY: 'space100',
+        },
+        MY_CUSTOM_ALERT_DISMISS_BUTTON: {
+          backgroundColor: 'colorBackgroundBodyInverse',
+          borderRadius: 'borderRadius20',
+        },
+      }}
+    >
+      <Alert element="MY_CUSTOM_ALERT" variant="neutral">
+        I am a custom neutral alert
+      </Alert>
+    </CustomizationProvider>
+  );
+};
+```
+
+## Find element identifiers
```

Comment:
I think it makes more sense to combine this section with the above section - the content here still feels like "understanding the element prop and data attributes" and the code examples in this section make more sense than the first example in the previous section. I'd get rid of that first example (I think it jumps into things too quickly, showing custom element values and the customization provider when the section is really just about understanding the element prop), and add another example at the end of this section, showing the same thing (coded HTML and rendered DOM) with a custom element name.